sormas-app/app/src/main/res/layout/fragment_person_edit_layout.xml

```diff
@@ -471,13 +471,20 @@
 
                 <de.symeda.sormas.app.component.controls.ControlTextEditField
                     android:id="@+id/person_fathersName"
-                    android:nextFocusForward="@+id/person_phone"
+                    android:nextFocusForward="@+id/person_nameOfOtherGuardians"
                     app:value="@={data.fathersName}"
                     app:imeOptions="actionNext"
                     style="@style/ControlSecondOfTwoColumnsStyle" />
 
             </LinearLayout>
 
+            <de.symeda.sormas.app.component.controls.ControlTextEditField
```

Comment:
As you probably know, this field also shows in for apps of non-german servers.

Reply:
Will be hidden after I merge the salutation field branch.